tasks/main.yml

```diff
@@ -105,6 +112,8 @@
 - name: Ensure configured taps are tapped.
   homebrew_tap: "tap={{ item }} state=present"
   with_items: "{{ homebrew_taps }}"
+  become: yes
+  become_user: "{{ homebrew_user }}"
 
 # Cask.
 - name: Install configured cask applications.
```

Comment:
I think you'll need to add:

```
become: yes
become_user: "{{ homebrew_user }}"
```

to the tasks that install/uninstall Cask applications.